functions/PayrollProcessor.Functions/Features/Resources/ResourcesTrigger.cs

```diff
@@ -25,5 +32,48 @@ public class ResourcesTrigger
 
             return new OkResult();
         }
+
+        [FunctionName(nameof(CreateData))]
+        public async Task<ActionResult> CreateData(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "resources/data")] HttpRequest req,
+            [Table(Resource.Table.Employees)] CloudTable employeeTable,
+            [Table(Resource.Table.Payrolls)] CloudTable payrollsTable,
+            ILogger log)
+        {
+            log.LogInformation($"Creating all data: [{req}]");
+
+            var employeeSeed = new EmployeeSeed();
+
+            var domainSeed = new DomainSeed(employeeSeed);
+
+            foreach (var (employee, payrolls) in domainSeed.BuildAll(5, 10))
+            {
+                var employeeTableResult = await employeeTable.ExecuteAsync(TableOperation.Insert(EmployeeEntity.Map.From(employee)));
+
+                if (!(employeeTableResult.Result is EmployeeEntity employeeEntity))
+                {
+                    throw new Exception($"Could not save employee [{employee.Id}]: {employeeTableResult.HttpStatusCode} {employeeTableResult.Result}");
+                }
+
+                foreach (var grouping in payrolls.GroupBy(p => p.CheckDate.ToString("yyyyMMdd")))
```

Comment:
`CheckDate` is the partition key for the payrolls table, which means we need to insert batches grouped by the partition key.

Since the data is already internally consistent, we could do a `SelectMany()` for all employee payrolls and `GroupBy()` over that to create fewer insertions...

However, Table storage is super fast for insertions so it really doesn't matter.